Automattic/pocket-casts-android
# [Up Next Shuffle] Adding whats new for Shuffle feature

## Description
- This adds the whats new modal announcement for Shuffle feature
- Context: p1732666622838439-slack-C07T08CTND9
- The animation will run just once. See Dom's comment here: p1732728381890649/1732716309.660169-slack-C07T08CTND9

## Testing Instructions
1. Apply [whatsnew.patch](https://github.com/user-attachments/files/17938266/whatsnew.patch)
2. Open the app
3. See the whats new modal


## Screenshots or Screencast 
[once.webm](https://github.com/user-attachments/assets/32167262-382d-4374-b736-b2a58bf95791)


## Checklist
 
#### I have tested any UI changes...
